ui/src/overridableMapping.js

```diff
@@ -11,6 +11,7 @@ import {
   RightMenuItemMobile,
 } from './overridden/components/Menu';
 import { DocumentRequestFormFields } from './overridden/frontsite/DocumentRequest/DocumentRequestForm';
+import { NotAvailableCopyText } from './overridden/frontsite/Document/DocumentDetails/DocumentCirculation/NotAvailableCopyText';
```

Comment:
nit: I think it sounds more correct No instead of Not and the order οf the words as in the text below, I would go for  `NoCopyAvailableText`, but still not that important